testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1448,3 +1448,26 @@ def server_secret
   command = "sed -i '$d' /etc/hosts && sed -i '$d' /etc/hosts"
   $server.run(command)
 end
+
+# select an item from any dropdown
+And(/^I select "(.*?)" from "([^"]*)" dropdown/) do |selection, label|
+  # let the the select2js box filter open the hidden options
+  xpath_query = "//select[@name='#{label}']"
+  raise "xpath: #{xpath_query} not found" unless find(:xpath, xpath_query).click
+  # select the desired option
+  raise "#{label} #{selection} not found" unless find(:xpath, "//select[@name='#{label}']/option[contains(text(), '#{selection}')]").click
+end
+
+# check if a certain link is present in the channel mappings table
+Given(/^I see a "(.*?)" link in the table/) do |link_name|
```

Comment:
Also wondering for that one and https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/step_definitions/navigation_steps.rb#L616
Maybe there is a reason?

Reply:
If I just look for a link it can't find it. I have to look for a specific link within a table field apparently.

Reply:
There also is https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/step_definitions/navigation_steps.rb#L655
However I guess we can keep it that way, we might have a number of steps that we reinvent and we would have to do some cleanup at some point